code/modules/materials/materials.dm

```diff
@@ -0,0 +1,198 @@
+/datum/material
+	var/name = "material"
+	var/color = "#FFFFFF"
+	var/force_multiplier = 1
+	var/health_multiplier = 1
+	var/prefix = "material"
+	var/starting_material = 0
+	var/atom/my_atom
+	var/can_process = 0
+	var/conduction = 0
+	var/obj/item/stack/default_stack
+
+/datum/material/proc/fire_act()
+	return
+
+/datum/material/proc/throw_impact(atom/hit_atom)
+	return
+
+/datum/material/proc/init_material(var/atom/A)
+	return
+
+/datum/material/iron
+	name = "Iron"
+	color = "#908D8D"
+	force_multiplier = 1
+	health_multiplier = 1
+	prefix = "iron"
+	conduction = 1
+	default_stack = /obj/item/stack/sheet/metal
+
+/datum/material/gold
+	name = "Gold"
+	color = "" // Handled by the material init. proc.
+	force_multiplier = 0.25
+	health_multiplier = 0.25 // this aint digmineconstructcraft buddy
+	prefix = "gold"
+	conduction = 1
+	default_stack = /obj/item/stack/sheet/mineral/gold
+
```

Comment:
This should have the "unacidable" var, if you were yoinking the code from my EUK PR. It used gold to make things acid-proof.